### PR TITLE
chore: add convert to flux to vis dropdown

### LINKED ISSUE #2473: notebooks: Visualization panel should have a "convert to flux" option
~The "export to client library" option only makes sense for the panels that represent flux queries.  It shouldn't be an option on the visualization panel. I would only include it in the panels that have a "Convert to |> flux" option.~

We should add a "Convert to |> flux" option for viz panels.

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #2473

Adds "convert to flux" to visualization panel dropdown

### CODE REVIEW

File: src/flows/components/Sidebar.tsx [hunk 1]
```diff
@@ -182,7 +182,10 @@ const Sidebar: FC = () => {
               return true
             }
 
-            if (!/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family)) {
+            if (
+              !/^(inputs|transform)$/.test(PIPE_DEFINITIONS[type]?.family) &&
+              flow.data.byID[id].type !== 'visualization'
```
Comment (genehynson) [inline src/flows/components/Sidebar.tsx:187]: `type` is already defined above, right? We can change this to `type !== 'visualization'` which improves readability

Comment (genehynson) [inline src/flows/components/Sidebar.tsx:187]: also should this be `type === 'visualization'`? Do you have a gif of this working?

**Review by genehynson — CHANGES_REQUESTED** [review]
I tested these changes out and I don't think they actually build the query correctly. The visualization panels have the `.visual` query that would need to be appended to the `.source` query. Currently, only the `.source` query is being shown in the resulting flux script

Example: 
image


I believe it should be 
```
buckets()
|> limit(n: 100)
```

File: src/flows/components/Sidebar.tsx (hunk 1 above)
Comment (ShmuelLotman) [reply src/flows/components/Sidebar.tsx:187]: As commented below, it works, i guess tho we need to put in the .visual in there for it to fully function, i missed that it wasn't there when it converted to flux

Comment (ShmuelLotman) [reply src/flows/components/Sidebar.tsx:187]: Done

**Review by genehynson — APPROVED** [review]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2591)

**Review by genehynson — APPROVED** [review]
works great now! Thanks!

**Pull request merged**